### PR TITLE
Add support for Dimmer (SL)

### PR DESCRIPTION
**Pull request by Gophlin**
First ever contribution on GitHub, so be gentle :) 
Also not sure how to link this to my issue I created yesterday

### CODE REVIEW

**Comment by Gophlin** [comment]
@windkh , Can you please review this pull request and merge it?

**Comment by windkh** [comment]
Sure

File: README.md [hunk 1]
```diff
@@ -167,9 +170,43 @@ The output of the node is an array of status objects for every roller of the rol
 ]
 ```
 
+# Shelly Dimmer (SL) Node
+The node is able to turn on and turn off a shelly switch and set it's brightness. It outputs the status of all relays after every interaction with the shelly device.
```
Comment (windkh) [inline README.md:174]: it's --> its

File: shelly/99-shelly.js [hunk 2]
```diff
@@ -337,4 +337,103 @@ module.exports = function (RED) {
...
+                    else{
+                        onOff = "off"
+                    }
+                }
+
+                var brightness;
+                if(command.brightness !== undefined){
+                    if(command.brightness >=1 && command.brightness <= 100){
+                        brightness = command.brightness;
+                  }
```
Comment (windkh) [inline shelly/99-shelly.js:403]: The else is missing: what should happen here? Is that worth an error or why do we ignore it??

**Comment by Gophlin** [comment]
All set I think

**Comment by windkh** [comment]
Thanks, I will do a release asap

**Pull request merged**